tools/flang1/flang1exe/semfunc.c

```diff
@@ -5786,6 +5786,16 @@ ref_pd(SST *stktop, ITEM *list)
       XFR_ARGAST(1);
     }
     break;
+  case PD_isnan:
```

Comment:
I am assuming that support for isnan during initialization is deliberately left out. right?
real :: x = isnan(0.0)
F90-S-0155-Intrinsic not supported in initialization: isnan

Reply:
Yes, this is not supported.